src/Mono.Android/map.csv

```diff
@@ -1,29 +1,29 @@
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_DOUBLE_TAP,20,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_DOUBLE_TAP_AND_HOLD,40,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_SINGLE_TAP,19,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_SWIPE_DOWN,26,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_SWIPE_LEFT,27,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_SWIPE_RIGHT,28,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_SWIPE_UP,25,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_TRIPLE_TAP,21,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_3_FINGER_DOUBLE_TAP,23,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_3_FINGER_DOUBLE_TAP_AND_HOLD,41,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_3_FINGER_SINGLE_TAP,22,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_3_FINGER_SWIPE_DOWN,30,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_3_FINGER_SWIPE_LEFT,31,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_3_FINGER_SWIPE_RIGHT,32,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_3_FINGER_SWIPE_UP,29,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_3_FINGER_TRIPLE_TAP,24,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_4_FINGER_DOUBLE_TAP,38,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_4_FINGER_DOUBLE_TAP_AND_HOLD,42,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_4_FINGER_SINGLE_TAP,37,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_4_FINGER_SWIPE_DOWN,34,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_4_FINGER_SWIPE_LEFT,35,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_4_FINGER_SWIPE_RIGHT,36,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_4_FINGER_SWIPE_UP,33,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_4_FINGER_TRIPLE_TAP,39,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_DOUBLE_TAP,17,,,,
-?,30,android/accessibilityservice/AccessibilityService.GESTURE_DOUBLE_TAP_AND_HOLD,18,,,,
+E,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_DOUBLE_TAP,20,Android.AccessibilityServices.AccessibilityGesture,TwoFingerDoubleTap,remove,
```

Comment:
Silly Q, but does the `map2.csv`-parsing code in generator trim whitespace, such that we could make this "nicer" by adding spaces before/after the commas to make things line up "nicely"?  Or is that just "crazy"?

```
E,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_DOUBLE_TAP_AND_HOLD,40,Android.AccessibilityServices.AccessibilityGesture,TwoFingerDoubleTapAndHold,remove,
E,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_SINGLE_TAP         ,19,Android.AccessibilityServices.AccessibilityGesture,TwoFingerSingleTap       ,remove,
E,30,android/accessibilityservice/AccessibilityService.GESTURE_2_FINGER_SWIPE_DOWN         ,26,Android.AccessibilityServices.AccessibilityGesture,TwoFingerSwipeDown       ,remove,
```

Reply:
It does trim, but I don't think it's going to make things nicer to read.  You would have to pad to at least the size of the longest line which is 217 characters.

Reply:
You wouldn't necessarily need to pad the entire file, just each group of enums.  If you don't think it'll be nicer to read, though, there's no point.